rocketmq-spring-boot/src/main/java/org/apache/rocketmq/spring/support/RocketMQUtil.java

```diff
@@ -114,6 +114,8 @@ public static MessagingException convert(MQClientException e) {
 
         if (payloadObj instanceof String) {
             payloads = ((String) payloadObj).getBytes(Charset.forName(charset));
+        } else if (message.getPayload() instanceof byte[]) {
```

Comment:
It is better, IMO, to keep the code style same as line 115.

Reply:
payloadObj instanceof byte[] maybe better.

Reply:
done